.github/workflows/e2e.yml

```diff
@@ -37,7 +38,7 @@ jobs:
       - name: Build Uppy packages
         run: corepack yarn build
       - name: Run end-to-end browser tests
-        run: corepack yarn run e2e:ci
+        run: corepack yarn run e2e:ci --browser chrome --record --key ${{secrets.CYPRESS_RECORD_KEY}}
```

Comment:
@aduh95 this doesn't work unfortunately, this isn't appended after `e2e:headless`. Ideas how to fix while keeping the key in this file for the secret?

```json
    "e2e:ci": "start-server-and-test 'npm-run-all --parallel e2e:client start:companion' '1234|3020' e2e:headless",
    "e2e:headless": "yarn workspace e2e cypress:headless"
```

Reply:
```
❯ yarn e2e:ci -- --browser chrome
/Users/merlijn/code/uppy/node_modules/lazy-ass/index.js:117
      throw err;
      ^

Error: expected <NPM script name that starts server> <url or port> <NPM script name that runs tests>
example: start-test start 8080 test
see https://github.com/bahmutov/start-server-and-test#use

    at lazyAssLogic (/Users/merlijn/code/uppy/node_modules/lazy-ass/index.js:110:14)
    at lazyAss (/Users/merlijn/code/uppy/node_modules/lazy-ass/index.js:115:28)
    at Object.getArguments (/Users/merlijn/code/uppy/node_modules/start-server-and-test/src/utils.js:91:5)
    at Object.<anonymous> (/Users/merlijn/code/uppy/node_modules/start-server-and-test/src/bin/start.js:10:22)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Module._load (node:internal/modules/cjs/loader:827:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47

Node.js v18.0.0
```

Reply:
I don; t know the CLI API of `start-server-and-test`, but if it doesn't offer a workaround, there's not much we can but stop using it (or patch it).

Reply:
https://github.com/cypress-io/cypress/blob/63173a6315d06ffb9a7eeda17c2bad4baeab6875/cli/lib/cli.js#L119

It looks like you can use an env variable (`CYPRESS_RECORD_KEY`), which would be much more convenient in our case.
I'd be +1 to keep using the Electron browser as it looks simpler to setup and to replicate if we ever need to, and IIUC `--record` defaults to true anyway.